db/migrate/20150128155644_allow_null_for_adviser_covers_whole_of_uk.rb

```diff
@@ -0,0 +1,5 @@
+class AllowNullForAdviserCoversWholeOfUk < ActiveRecord::Migration
+  def change
+    change_column :advisers, :covers_whole_of_uk, :boolean, null: true, default: nil
```

Comment:
Spoke to @lshepstone irl about this and he clarified.  The column previously had a default value set, so this is reinstating the null / absence of a default.